.github/workflows/back-tests.yml

```
@@ -27,40 +27,46 @@ jobs:
     if: needs.changes-check.outputs.any_changed == 'true'
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v3
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)